src/components/Loader.js

```diff
@@ -75,14 +75,16 @@ Loader.propTypes = {
   color: React.PropTypes.string,
   isLoading: React.PropTypes.bool,
   isRelative: React.PropTypes.bool,
-  isSmall: React.PropTypes.bool
+  isSmall: React.PropTypes.bool,
+  loaderText: React.PropTypes.string
```

Comment:
Since we already have the context of the component name, `<Loader>`, I think we can rename this to just `text`, or better yet we can use `this.props.children` and let the user pass in whatever they want (text, numbers, jsx).